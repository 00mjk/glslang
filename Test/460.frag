#version 460 core

struct S {
    float f;
    vec4 v;
};

in S s;

void main()
{
    interpolateAtCentroid(s.v);
}
