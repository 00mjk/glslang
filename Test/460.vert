#version 460 core

int i;
; // extraneous semicolon okay
float f;;;

void main()
{
}
;
;
