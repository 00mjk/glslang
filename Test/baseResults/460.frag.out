460.frag
Shader version: 460
0:? Sequence
0:10  Function Definition: main( ( global void)
0:10    Function Parameters: 
0:12    Sequence
0:12      interpolateAtCentroid ( global 4-component vector of float)
0:12        v: direct index for structure ( global 4-component vector of float)
0:12          's' ( smooth in structure{ global float f,  global 4-component vector of float v})
0:12          Constant:
0:12            1 (const int)
0:?   Linker Objects
0:?     's' ( smooth in structure{ global float f,  global 4-component vector of float v})


Linked fragment stage:


Shader version: 460
0:? Sequence
0:10  Function Definition: main( ( global void)
0:10    Function Parameters: 
0:12    Sequence
0:12      interpolateAtCentroid ( global 4-component vector of float)
0:12        v: direct index for structure ( global 4-component vector of float)
0:12          's' ( smooth in structure{ global float f,  global 4-component vector of float v})
0:12          Constant:
0:12            1 (const int)
0:?   Linker Objects
0:?     's' ( smooth in structure{ global float f,  global 4-component vector of float v})

